README.md

```diff
@@ -1,2 +1,52 @@
-# jupyterhub-kubernetes-spawner
-Kubernetes spawner for JupyterHub
+# jupyterhub-kubernetes-spawner #
+
+The KubernetesSpawner enables JupyterHub to spawn singleuser notebooks on a [kubernetes](https://kubernetes.io/) cluster.
+
+## Features ##
+
+Kubernetes is a popular & powerful container orchestration platform that helps run containerized applications at scale. If you want to run a JupyterHub setup that needs to scale across multiple nodes (anything with over ~50 simultaneous users), Kubernetes is a wonderful way to do it.
```

Comment:
Perhaps use from google's site:

Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications.